test/src/test/java/org/corfudb/integration/CorfuReplicationClusterConfigIT.java

```diff
@@ -1007,6 +1011,140 @@ public void testActiveLockRelease() throws Exception {
+            mapActive.put(String.valueOf(i), i);
+            activeRuntime.getObjectsView().TXEnd();
+        }
+        assertThat(mapActive.size()).isEqualTo(largeBatch);
+
+        // Write 10 entries to backup map
+        for (int i = 0; i < firstBatch; i++) {
+            backupRuntime.getObjectsView().TXBegin();
+            mapBackup.put(String.valueOf(i), i);
+            backupRuntime.getObjectsView().TXEnd();
+        }
+
+        // Wait until data is fully replicated again
+        waitForReplication(size -> size == largeBatch, mapStandby, largeBatch);
+        log.info("After delta sync, both maps have size {}. Current active corfu[{}] log tail " +
+                        "is {}, standby corfu[{}] log tail is {}", secondBatch, activeClusterCorfuPort,
+                activeRuntime.getAddressSpaceView().getLogTail(), standbyClusterCorfuPort,
+                standbyRuntime.getAddressSpaceView().getLogTail());
+
+        // Verify data
+        for (int i = 0; i < largeBatch; i++) {
+            assertThat(mapStandby.containsKey(String.valueOf(i))).isTrue();
+        }
+
+        // Change the topology - brings up the backup cluster
+        try (TxnContext txn = activeCorfuStore.txn(DefaultClusterManager.CONFIG_NAMESPACE)) {
+            txn.putRecord(configTable, DefaultClusterManager.OP_BACKUP, DefaultClusterManager.OP_BACKUP, DefaultClusterManager.OP_BACKUP);
+            txn.commit();
+        }
+        log.info("Change the topology!!!");
+
+        TimeUnit.SECONDS.sleep(shortInterval);
+
+
+        // Perform an enforce full snapshot sync
+        try (TxnContext txn = activeCorfuStore.txn(DefaultClusterManager.CONFIG_NAMESPACE)) {
+            txn.putRecord(configTable, DefaultClusterManager.OP_ENFORCE_SNAPSHOT_FULL_SYNC,
+                    DefaultClusterManager.OP_ENFORCE_SNAPSHOT_FULL_SYNC, DefaultClusterManager.OP_ENFORCE_SNAPSHOT_FULL_SYNC);
+            txn.commit();
+        }
+        TimeUnit.SECONDS.sleep(mediumInterval);
+
+        // Standby map size should be 10
+        waitForReplication(size -> size == firstBatch, mapStandby, firstBatch);
+
+        // Write 5 entries to backup map
+        for (int i = firstBatch; i < secondBatch; i++) {
+            backupRuntime.getObjectsView().TXBegin();
+            mapBackup.put(String.valueOf(i), i);
+            backupRuntime.getObjectsView().TXEnd();
+        }
+        assertThat(mapBackup.size()).isEqualTo(secondBatch);
+
+        // Wait until data is fully replicated again
+        waitForReplication(size -> size == secondBatch, mapStandby, secondBatch);
+        log.info("After delta sync, both maps have size {}. Current backup corfu[{}] log tail " +
+                        "is {}, standby corfu[{}] log tail is {}", firstBatch, backupClusterCorfuPort,
+                backupRuntime.getAddressSpaceView().getLogTail(), standbyClusterCorfuPort,
+                standbyRuntime.getAddressSpaceView().getLogTail());
+
+        shutdownCorfuServer(backupCorfu);
```

Comment:
Should you shutdown active and standby as well?

Reply:
Yes, they are shutdown by the @After function